aaelf64/aaelf64.rst

```diff
@@ -1635,6 +1639,8 @@ The dynamic relocations for those execution environments that support only a lim
 
 With the exception of ``R_<CLS>_COPY`` all dynamic relocations require that the place being relocated is an 8-byte aligned 64-bit data location in ELF64 or a 4-byte aligned 32-bit data location in ELF32.
 
+``R_<CLS>_ABS64`` and ``R_<CLS>_ABS32`` may only appear in a well-formed executable or dynamic shared object in ELF64 or ELF32 respectively.  Note that for their respective file format these relocations are both static and dynamic relocations.
```

Comment:
Looks like there are two spaces between "respectively.  Note" not sure if this will show up in the rendering, but probably worth fixing if non-intentional.

Reply:
Heh -- GNU comment style creeping in, will fix ;-)